src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -230,6 +231,32 @@ def test_wraps_the_corresponding_method_of_the_held_interface(
         assert manager.isEntityReferenceString(a_ref_string) == expected
         method.assert_called_once_with(a_ref_string, a_host_session)
 
+    @pytest.mark.parametrize(
+        # Override `manager_info_dict` fixture here, so that it is used
+        # when the `manager` fixture is constructed for the test case.
+        "manager_info_dict,expected",
+        (
+            ({constants.kInfoKey_EntityReferencesMatchPrefix: "asset://"}, True),
```

Comment:
[fix] May we sneak some unicode in there?

Reply:
Added (force-pushed). 

As an aside, I tried to find a UTF-8 character such that when a byte was truncated it would become another valid character -  obviously a problem for a prefix match based on `char` bytes.  That led me to discover a little bit about UTF-8 encoding and the fact that such an issue is impossible (assuming valid UTF-8 codepoints in both strings).